.github/workflows/doc_conda.yml

```diff
@@ -0,0 +1,47 @@
+name: HTTomo doc test
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+      - documentation
+      - docs
+  push:
+    branches:
+      - main
+      - documentation
+      - docs
```

Comment:
since we are merging this to `main`, we don't have to run the workflow on these branches...

Reply:
yep, please correct that?